src/action.c

```diff
@@ -692,6 +765,21 @@ actions_run(struct view *activator, struct server *server,
 				desktop_focus_output(output_from_name(server, output_name));
 			}
 			break;
+		case ACTION_TYPE_IF:
+			if (view) {
+				run_if_action(view, server, action);
+			}
+			break;
+		case ACTION_TYPE_FOR_EACH:
+			struct wl_array views;
```

Comment:
`;` or `{` as first statement here please (older compilers and stuff).

Reply:
Agree, done.